pkg/manifest/os.go

```diff
@@ -287,6 +287,21 @@ func (p *OS) getContainerSources() []container.SourceSpec {
 	return p.OSCustomizations.Containers
 }
 
+func tomlPkgsFor(distro Distro) []string {
+	switch distro {
+	case DISTRO_EL7:
+		// Supported by upstream but superseeded by tomllib in stdlib
+		return []string{"python3-toml"}
+	case DISTRO_EL8:
+		// old, unmaintained lib
+		return []string{"python3-pytoml"}
+	default:
+		// No extra package needed, on rhel10 and
+		// fedora. stdlib has "tomlib"
+		return nil
+	}
+}
+
```

Comment:
I think we have a pre-existing issue here that was never caught, because we don't cover it with a build config.

Both stages that trigger this package selection, `org.osbuild.containers.storage.conf` and `org.osbuild.bootc.install.config`, need to write toml files.  In EL10, this will require `python3-tomli-w`.

Reply:
Thanks, I updated this too now.